config/scim2-parent-unit-checkstyle.xml

```diff
@@ -162,5 +167,18 @@
     <!-- Ensure that all long constants are followed by a capital L. -->
     <module name="UpperEll" />
   </module>
-</module>
 
+  <!-- Ensure that there are no lines longer than 120 characters in the test
+       code. -->
+  <module name="LineLength">
+    <property name="fileExtensions" value="java"/>
+    <property name="max" value="120"/>
```

Comment:
In `config/scim2-parent-checkstyle.xml`, this max value is set to 100. Is it worth changing either this or the other value so they match each other?

Reply:
I left this higher because the line-length styling is much less important in the test code in my opinion. It seems okay to give more leniency than 100 characters in the test package. But let me know if you disagree.

Reply:
Sure, that sounds fine to me.